pkg/devfile/generator/generators.go

```diff
@@ -187,6 +187,10 @@ func GetDeployment(devfileObj parser.DevfileObj, deployParams DeploymentParams)
 		Spec:       *getDeploymentSpec(deploySpecParams),
 	}
 
+	if deployParams.Replicas != nil {
+		deployment.Spec.Replicas = deployParams.Replicas
```

Comment:
I think that this affectation should be done in getDeploymentSpec which is the dedicated function for this.

Reply:
I agree. That's where I first looked. But that would mean changing the signature of `getDeploymentSpec`. I was not sure if that would be OK. But it's an internal function, so changing signature shouldn't be a big deal.

Pushed https://github.com/devfile/library/pull/129/commits/4a544b12a31ed89f58f1da0e08fa074ab203726a. PTAL.